runtime/near-runtime-fees/src/lib.rs

```diff
@@ -74,10 +76,12 @@ pub struct RuntimeFeesConfig {
     pub pessimistic_gas_price_inflation_ratio: Rational,
 
     /// Describes cost of running method of evm, include deploy code and call contract function
+    #[cfg(feature = "protocol_feature_evm")]
     pub evm_config: EvmCostConfig,
 
     /// New EVM deposit.
     /// Fee to create new EVM account.
+    #[cfg(feature = "protocol_feature_evm")]
     #[serde(with = "u128_dec_format", default = "default_evm_deposit")]
```

Comment:
I think you can use constants in default, so there is no need for `default_evm_deposit`

Reply:
Doesn't work:
```
error[E0618]: expected function, found `u128`
  --> runtime/near-runtime-fees/src/lib.rs:85:49
   |
15 | const EVM_DEPOSIT: Balance = 1_000_000_000_000_000_000_000_000_000;
   | ------------------------------------------------------------------- `u128` defined here
...
57 | #[derive(Debug, Serialize, Deserialize, Clone, Hash, PartialEq, Eq)]
   |                            ----------- call expression requires function
...
85 |     #[serde(with = "u128_dec_format", default = "EVM_DEPOSIT")]
```